dffml/operation/preprocess.py

```diff
@@ -60,3 +60,16 @@ async def literal_eval(str_to_eval: str):
     """
     value = ast.literal_eval(str_to_eval)
     return {"str_after_eval": value}
+
+
+edit_feature_def = Definition(name="feature_data", primitive="generic")
+
+
+@op(
+    name="edit_feature",
+    inputs={"features": edit_feature_def},
+    outputs={"updated_features": edit_feature_def},
+)
+async def edit_feature(features):
```

Comment:
Looking forward to the mnist example, we should make it so that if `multiplicand`  is an array, it multiplies every element of the array by the `multiplier` (so that you can do normalization) include an example of this in the docstring example

Reply:
I was thinking that the operation can be modified in the following way to accommodate other mathematical operations :

```python
edit_feature_def = Definition(name="feature_data", primitive="generic")
operator_def = Definition(name="operator_sign", primitive="str")
factor_def = Definition(name="factor_def", primitive="generic")

@op(
   name="edit_feature",
   inputs={"features": edit_feature_def, "operator": operator_def, "factor": factor_def},
   outputs={"updated_features": edit_feature_def},
)
async def edit_feature(features, operator, factor):
   operators = {
      "+": features + factor,
      "-": features - factor,
      "*": features * factor,
      "/": features / factor,
   }

   value = operators[operator]
   return {"updated_features": value}
```

Reply:
The issue I have with this is that operations aren't necessarily tied to feature modification. Naming one `edit_feature` doesn't keep with the generic nature of things, since we might want to multiply something that's not related to ML or features in some dataflow.

Reply:
Also, the above code computes three unused results